PL2/PL2-core/src/test/java/nl/tudelft/pl2016gr2/core/algorithms/FilterSnipsTest.java

```diff
@@ -27,7 +27,7 @@
   @Before
   public void setup() {
     for (int i = 1; i <= 9; i++) {
-      Node node = new Node(i, 1, null, 0);
+      Node node = new Node(i, 1, new ArrayList<>(), 0);
       if (i == 1 || i == 4 || i == 7) {
```

Comment:
if (i - 1 mod 3 == 0) kan je doen, dan is het duidelijk dat het om elke 3 nodes vanaf 1 gaat en heb je geen magic numbers meer. Weet niet of checkstyle dat ook in de tests bekijkt